core/resources/shaders/sdf.fs

```diff
@@ -68,16 +69,20 @@ float sampleAlpha(in vec2 uv, float distance, float threshold) {
 
 void main(void) {
 
-    float threshold_fill = 0.5;
-    float threshold_stroke = threshold_fill - v_strokeWidth * u_device_pixel_ratio;
-
     float distance = texture2D(u_tex, v_texcoords).a;
-
-    float alpha_fill = pow(sampleAlpha(v_texcoords, distance, threshold_fill), 0.4545);
-    float alpha_stroke = pow(sampleAlpha(v_texcoords, distance, threshold_stroke), 0.4545);
-
-    vec4 color = mix(v_strokeColor, v_color, alpha_fill);
-    color.a = max(alpha_fill, alpha_stroke) * v_alpha * v_color.a;
+    vec4 color;
+
+    if (u_pass == 1) {
```

Comment:
Branching on uniforms is quite cheap as far as I know, as it should be easy to predict which branch to take. Both branches must only be computed when both can be reached within a shader unit (those that are processing e.g. 16x16 pixel in parallel with one process counter). That's just guessing though, but may the reason why apple states that branching on uniforms is acceptable: https://developer.apple.com/library/ios/documentation/3DDrawing/Conceptual/OpenGLES_ProgrammingGuide/BestPracticesforShaders/BestPracticesforShaders.html

I'll see if it is possible to move the branching to the vertex shader